Ghidra/Features/VersionTracking/src/test.slow/java/ghidra/feature/vt/gui/provider/VTFunctionReferenceCorrelator_ELF_Test.java

```diff
@@ -105,7 +105,7 @@ public void testFunctionReferenceCorrelator_allFunctionMatches() throws Exceptio
 		List<VTMatchSet> exactMatchSets = session.getMatchSets();
 		for (VTMatchSet ms : exactMatchSets) {
 			String corrName = ms.getProgramCorrelatorInfo().getName();
-			if (corrName == exactSymbolNameCorrelator) {
+			if (corrName.equals(exactSymbolNameCorrelator)) {
```

Comment:
swap the 2 string variables around so its exactSymbolNameCorrelator.equals(corrName) and you can make it a little more bullet proof.

Reply:
You are surely right, thank you!